firebase-auth/src/androidMain/kotlin/dev/gitlive/firebase/auth/credentials.kt

```diff
@@ -52,17 +51,17 @@ public actual object GoogleAuthProvider {
     }
 }
 
-public val OAuthProvider.android: OAuthProvider get() = android
+public val OAuthProvider.android: com.google.firebase.auth.OAuthProvider get() = android
```

Comment:
To not have to add this everywhere, you could consider using `import com.google.firebase.auth.OAuthProvider as AndroidOAuthProvider`